webpack.config.prod.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-undef */
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+const Dotenv = require('dotenv-webpack');
+
+process.env.NODE_ENV = 'production';
+
+module.exports = {
+	mode: 'production',
+	target: 'web',
+	devtool: 'source-map',
+	entry: './src/index',
+	output: {
+		path: __dirname + '/public/',
+		publicPath: '/',
+		filename: 'bundle.js',
+		chunkFilename: '[name].js',
+	},
+	plugins: [
+		new Dotenv({
+			path: path.resolve(__dirname, './.env'),
+			safe: true,
+			systemvars: true,
+			silent: true,
+			defaults: false,
+		}),
+		new BundleAnalyzerPlugin({ analyzerMode: 'static' }),
+		new HtmlWebpackPlugin({
+			template: __dirname + '/public/index.html',
+			minify: {
+				removeComments: true,
+				collapseWhitespace: true,
+				removeRedundantAttributes: true,
+				useShortDoctype: true,
+				removeEmptyAttributes: true,
+				removeStyleLinkTypeAttributes: true,
+				keepClosingSlash: true,
+				minifyJS: true,
+				minifyCSS: true,
+				minifyURLs: true,
+			},
```

Comment:
comments are required, whitespace is required too for easy maintainability.
consider this:
```
new HtmlWebpackPlugin({
      template: path.resolve('./public/index.html'),
      inject: true
    }),
```
this will be very enough